src/apps/app_webhook.erl

```diff
@@ -0,0 +1,126 @@
+-module(app_webhook).
+-export([start/1, new_block/2, new_transaction/2]).
+-include("ar.hrl").
+-include("ar_config.hrl").
+
+-define(NUMBER_OF_TRIES, 10).
+-define(WAIT_BETWEEN_TRIES, 30 * 1000).
+
+-define(BASE_HEADERS, [
+	{<<"content-type">>, <<"application/json">>}
+]).
+
+start(Configs) ->
+	adt_simple:start(?MODULE, Configs).
+
+new_block(Configs, Block) ->
+	call_webhooks(Block, Configs),
+	Configs.
+
+new_transaction(Configs, Tx) ->
+	call_webhooks(Tx, Configs),
+	Configs.
+
+call_webhooks(Entity, Configs) ->
```

Comment:
What do you think about

```
call_webhooks({transaction, TX}, Configs),
```

and later

```
call_webhooks({Event, Entity}, Configs) ->
...
case lists:member(Event, Events) of
...
```
?

Reply:
Addressed in 02d5ac19 👍